README.md

```diff
@@ -78,10 +78,18 @@ they don't violate our quality standards. To avoid frustration, before
 sending us your pull request please run full Maven build:
 
 ```
-$ mvn clean install -Pqulice
+$ mvn clean install -Pqulice -PDockerITs
```

Comment:
@mkordas this was basically just a usability choice. You really need to have your Docker setup cleanly installed locally for everything to work fine.
There are a million random things you can do wrong here (not Rultor's job to document this in the Readme ... any competent Docker user will be able to figure this out ...).
Realistically speaking these tests will run correctly for a lot fewer users than they'll work fine for (without Linux especially this does take some fiddling to get right). So I went with the majority for the default.
Also ... otherwise I'll have to do another 2 step merge to make Rultor happy, since the command without this profile is still in the `.rultor.yml` and Rultor cannot run the Docker tests, since it runs inside a Container and `privileged` is not an option for now :/